oauth2client/flask_util.py

```diff
@@ -191,6 +192,21 @@ def requires_calendar():
 __author__ = 'jonwayne@google.com (Jon Wayne Parrott)'
 
 DEFAULT_SCOPES = ('email',)
+CREDENTIALS_KEY = 'google_oauth2_credentials'
+FLOW_KEY = 'google_oauth2_flow_{0}'
+CSRF_KEY = 'google_oauth2_csrf_token'
+
+
+def _get_flow_for_token(csrf_token):
+    """Retrieves the flow instance associated with a given CSRF token from
+    the Flask session."""
+    flow_pickle = session.get(
+        FLOW_KEY.format(csrf_token), None)
+
+    if flow_pickle is None:
```

Comment:
Eh, that toes the line of being more terse than readable. I'd prefer not to, but if you feel strongly I'll change it.